src/test/java/io/aiven/kafka/connect/common/config/AivenCommonConfigTest.java

```diff
@@ -16,6 +16,7 @@
 
 package io.aiven.kafka.connect.common.config;
 
+import java.util.HashMap;
 import java.util.Map;
```

Comment:
```suggestion

```

Reply:
`Map` is still used on line 34. I think the linter is still on and detects this on the build.